packages/devkit/devkit.1.20210120/opam

```diff
@@ -7,12 +7,12 @@ dev-repo: "git+https://github.com/ahrefs/devkit.git"
 bug-reports: "https://github.com/ahrefs/devkit/issues"
 synopsis: "Development kit - general purpose library"
 build: [
-  ["dune" "subst"] {pinned}
+  ["dune" "subst"] {dev}
```

Comment:
I understand `dev` subsumes `pinned` so I should replace in opam file in devkit git repo, right?

Reply:
yes please!